backend/backend/settings.py

```diff
@@ -183,19 +194,36 @@
         "rest_framework.authentication.TokenAuthentication",
     ],
     "EXCEPTION_HANDLER": "backend.exception_handler.bad_request_logger",
+    "DEFAULT_RENDERER_CLASSES": (
+        "djangorestframework_camel_case.render.CamelCaseJSONRenderer",
+        "djangorestframework_camel_case.render.CamelCaseBrowsableAPIRenderer",
+    ),
+    "DEFAULT_PARSER_CLASSES": (
+        "djangorestframework_camel_case.parser.CamelCaseFormParser",
+        "djangorestframework_camel_case.parser.CamelCaseMultiPartParser",
+        "djangorestframework_camel_case.parser.CamelCaseJSONParser",
+    ),
 }
 
+# MARK: Spectacular
+
 SPECTACULAR_SETTINGS = {
     "TITLE": "activist.org API",
-    "DESCRIPTION": "Open-source, nonprofit activism platform",
+    "DESCRIPTION": "An open-source activism platform",
     "VERSION": "0.1.0",
     "SERVE_INCLUDE_SCHEMA": False,
+    "CAMELIZE_NAMES": False,
```

Comment:
I see another comment where you noted you were still getting snake_case responses.  Could this have something to do with it?

Reply:
Good check :) This is specifically for the table names within the Swagger UI, not the results of the backend itself. Keeping the table names snake_case makes sense to me as they should have consistent identifiers.